Cisco Meraki metrics package [WIP] (#11069)
elastic/integrations: packages/cisco_meraki_metrics/_dev/build/docs/README.md

# Cisco Meraki Metrics Integration

This integration periodically fetches metrics from [Cisco Meraki](https://meraki.cisco.com/) networks. It fetches a wide range of metrics including device details and status, network performance measurements, and switch port information. The integration also collects metrics about wireless channel use and uplink performance.

These metrics help you understand how well your Meraki network is working and make it easier to monitor and manage your network setup.

## Compatibility

The integration uses the [Meraki Dashboard RESTFul APIs](https://github.com/meraki/dashboard-api-go/) library to collect metrics from Cisco Meraki networks.

## Requirements

You need Elasticsearch for storing and searching your data and Kibana for visualizing and managing it.
You can use our hosted Elasticsearch Service on Elastic Cloud, which is recommended, or self-manage the Elastic Stack on your own hardware.

> ## Configuration

Reviewer: Steps for configuring missing here

# Cisco Meraki Metrics Integration

> This integration periodically fetches metrics from [Cisco Meraki](https://meraki.cisco.com/) networks. It fetches a wide range of metrics including device details and status, network performance measurements, and switch port information. The integration also collects metrics about wireless channel use and uplink performance.

Reviewer 2: ```suggestion
This integration periodically fetches metrics from [Cisco Meraki](https://meraki.cisco.com/) networks. It collects a wide range of metrics including device details and status, network performance measurements, switch port information, wireless channel utilization, and uplink performance.
```

These metrics help you understand how well your Meraki network is working and make it easier to monitor and manage your network setup.

## Compatibility

> The integration uses the [Meraki Dashboard RESTFul APIs](https://github.com/meraki/dashboard-api-go/) library to collect metrics from Cisco Meraki networks.

Reviewer 2: ++. Especially the meraki dashboard api version used

> The integration uses the [Meraki Dashboard RESTFul APIs](https://github.com/meraki/dashboard-api-go/) library to collect metrics from Cisco Meraki networks.

Author: stack version is in the package manifest. there's only one version of the API - and it's accessed via the SDK, so no future compatibility issues there.

## Requirements

> You need Elasticsearch for storing and searching your data and Kibana for visualizing and managing it.
> You can use our hosted Elasticsearch Service on Elastic Cloud, which is recommended, or self-manage the Elastic Stack on your own hardware.

Reviewer 2: ```suggestion
To use this integration, you need:

- Elasticsearch for storing and searching your data
- Kibana for visualizing and managing your data

You can use our hosted Elasticsearch Service on Elastic Cloud (recommended) or self-manage the Elastic Stack on your own hardware.
```

Reviewer 3: This is copied from other data streams docs, most of them have it like that.

## Configuration

> You need the following information from `Cisco Meraki` to configure this integration in Elastic:
> 
> - API Key
> - Organization IDs
> - API Base URL (optional)
> 
> You can find more information about the required settings for this integration in the [Meraki Dashboard API documentation](https://documentation.meraki.com/General_Administration/Other_Topics/Cisco_Meraki_Dashboard_API).

Reviewer 2: ```suggestion
To configure this integration in Elastic, you need the following information from `Cisco Meraki`:

- API Key
- Organization IDs
- API Base URL (optional)

For more details on these settings, refer to the [Meraki Dashboard API documentation](https://documentation.meraki.com/General_Administration/Other_Topics/Cisco_Meraki_Dashboard_API).
```

### Enabling the integration in Elastic

> 1. In Kibana go to **Management > Integrations**
> 2. In "Search for integrations" search bar type **Meraki**
> 3. Click on "Cisco Meraki metrics" integration from the search results.
> 4. Click on **Add Cisco Meraki metrics Integration** button to add the integration.

Reviewer 2: ```suggestion
1. In Kibana, navigate to **Management > Integrations**
2. In the "Search for integrations" search bar, type **Meraki**
3. Click on "Cisco Meraki metrics" integration from the search results
4. Click on the **Add Cisco Meraki metrics Integration** button to add the integration
```
